app/src/main/java/de/westnordost/streetcomplete/quests/building_levels/AddBuildingLevels.kt

```diff
@@ -17,7 +17,7 @@ class AddBuildingLevels(o: OverpassMapDataDao) : SimpleOverpassQuestType<Buildin
             "commercial","office","warehouse","industrial","manufacture","parking","farm",
             "farm_auxiliary","barn","cabin").joinToString("|") + "\n" +
         " and !building:levels and !height and !building:height\n " +
-        " and !man_made and location!=underground\n "
+        " and !man_made and location!=underground and ruins!=yes\n "
```

Comment:
Why not this?

Reply:
https://www.openstreetmap.org/way/131358967 is an extreme example and one of things that prompted my patch eliminating ruins=yes - just wall part remaining from something that was ancient marketplace/mall and SC will now happily create quest about it. 

And what worse there is no tagging that will get rid of it, in this case even adding historic=* will not help and given no access to it is not viable to tag with `height` (maybe there is some official data/3D model on a suitable licence?).

But even without such ridiculous cases - there are often building in advanced decay where floor count is far from obvious and wasting time on tagging them is likely to be pointless.

Reply:
I'd say that floor count is the **current** floor count, not one that might have been one time.

[But looking at the site on google maps 3d](https://www.google.de/maps/@41.891174,12.4867434,51a,35y,82.29h,61.68t/data=!3m1!1e3), I understand the problem.

Basically, I'd say the tag is wrong - it is not a building. It is a piece of wall. But I guess there are no correct tags for this maybe? Anyway, I rest my case. Let's exclude ruins from these quests.